src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_postgres_scenario.py

```diff
@@ -0,0 +1,214 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+from azure.cli.testsdk import JMESPathCheck, ScenarioTest, ResourceGroupPreparer
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+
+class CosmosDBPostgresScenarioTest(ScenarioTest):
+
+    # @AllowLargeResponse()
+    # @ResourceGroupPreparer(random_name_length=17, name_prefix='cli_test')
+    # def test_cosmosdb_postgres_create_cluster_with_configurations(self, resource_group):
```

Comment:
Why comment out these tests? I noticed that these tests' recording files are still committed...

Reply:
@evelyn-ys
Hi Yishi, sorry about the leftover comments last time. I've cleaned up the code now and re-run all tests - could you give it another look? Thanks!